From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF(TST): allow AttributeError being raised (in addition to TypeError) from mpl
  origin:  https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=827938

--- a/pandas/tests/test_graphics.py
+++ b/pandas/tests/test_graphics.py
@@ -1318,7 +1318,8 @@ class TestDataFramePlots(TestPlotBase):
         self._check_axes_shape(axes, axes_num=4, layout=(4, 1))
 
         df = DataFrame({'x': [1, 2], 'y': [3, 4]})
-        with tm.assertRaises(TypeError):
+        # mpl >= 1.5.2 (or slightly below) throw AttributError
+        with tm.assertRaises((TypeError, AttributeError)):
             df.plot.line(blarg=True)
 
         df = DataFrame(np.random.rand(10, 3),
